tools/mOTUs/motus_profiler.xml

```diff
@@ -0,0 +1,254 @@
+        -t \${GALAXY_SLOTS:-8} 
+        -n '$sample_name' 
+        -db '$DATABASE_DIR'
+        #if $inputs.input_type.selector == 'reads'
+            #if $inputs.input_type.reads_type.selector == 'paired'
+                -f '${inputs.input_type.reads_type.forward_reads}' 
+                -r '${inputs.input_type.reads_type.reverse_reads}'
+            #elif $inputs.input_type.reads_type.selector == 'single'
+                -s '${inputs.input_type.reads_type.unpaired_reads}'   
+         
+            #elif $input_type.selector == 'SAM_BAM_input'
+                -i '${inputs.input_type.SAM_BAM_input}'
+            #elif $input_type.selector == 'mgc_read_count'
+                -m '${inputs.input_type.mgc_read_count}'
+            #end if
+        #end if
+        #if $algorithm_advanced_options.advanced_options.customize_algorithm_options == 'customize'
+            -g $algorithm_advanced_options.advanced_options.marker_genes_cutoff
+            -l $algorithm_advanced_options.advanced_options.minimum_alignment_len
+            -v $algorithm_advanced_options.advanced_options.verbosity_level_selection
+            -y $algorithm_advanced_options.advanced_options.read_count_type_selection
+        #end if
+        #if $additional_options.advanced == "advanced"
+            #if $additional_options.ref_motus_only
+                -e 
+            #end if
+            #if $additional_options.full_species_name
+                -u
+            #end if
+            #if $additional_options.result_as_counts
+                -c
+            #end if
+            #if $additional_options.ncbi_taxonomy_ids
+                -p
+            #end if
+            #if $additional_options.biom_format
+                -B
+            #end if
+            #if $additional_options.cami_format
+                -C
+            #end if
+            #if $additional_options.full_rank_taxonomy
+                -q
+            #end if
+            #if $additional_options.taxonomic_level_metaphlanType
+                -A
+            #end if
+            #if $additional_options.taxonomic_level
+                -k
+            #end if
+        #end if
+        > '${mOTUs_profile}'
+    ]]></command>
+    <inputs>
+        <section name="inputs" title="Inputs" expanded="true">
+            <conditional name="input_type">
+                <param name="selector" type="select" label="Input file(s)">
+                    <option value="reads" selected="True">fastq,fastq.gz file(s)</option>
+                    <option value="SAM_BAM_input" selected="true">SAM/BAM input file(s) generated by motus map_tax</option>
+                    <option value="mgc_read_count" selected="True">mgc_read_count file(s) generated by motus cal-mgc</option>
+                </param>
+                <when value="reads">
+                    <conditional name="reads_type">
+                        <param name="selector" type="select" format="fastq,fastq.gz" label="Choose input file(s)">
```

Comment:
a selector has not format afaik, are you using the Galaxy language server? It should raise hopefully a warning